question.py

```diff
@@ -1,57 +1,58 @@
 import random
 from fractions import Fraction
 
+
 class Question:
     def __init__(self):
         self.left_side = 0
         self.right_side = 0
         self.answer = 0
         next(self)
-        
 
     def __next__(self):
         self.populate_multiple_choice()
 
     def populate_multiple_choice(self):
         self.answer_index = random.randrange(4)
         self.choices = []
-        
-        for num in range(0,4):
+
+        for num in range(0, 4):
             if num == self.answer_index:
                 self.left_side = self.get_fraction()
                 self.right_side = self.get_fraction()
                 self.answer = self.left_side + self.right_side
                 while (
-                    any([x == self.answer for x in self.choices]) 
+                    any([x == self.answer for x in self.choices])
                 ):
```

Comment:
The indentation here is off and the parentheses covering the any isn't needed too, the indentation is off as the any is same indent as the body of the loop so it's confusing.